VMBackup/main/Utils/HandlerUtil.py

```diff
@@ -154,6 +164,7 @@ def log(self, message,level='Info'):
         except Exception as e:
             try:
                 errMsg='Exception in hutil.log'
+                self.log_with_no_try_except(e, 'Warning')
                 self.log_with_no_try_except(errMsg, 'Warning')
```

Comment:
If we are logging exception. L168 can be skipped if we append the message to L167.